jhotdraw-core/src/main/java/org/jhotdraw/draw/figure/BezierFigure.java

```diff
@@ -171,11 +171,12 @@ protected void drawFill(Graphics2D g) {
 
   @Override
   public boolean contains(Point2D.Double p, double scaleDenominator) {
+    boolean result = false;
     double tolerance =
         Math.max(1f, 2 * AttributeKeys.getPerpendicularHitGrowth(this, scaleDenominator));
     if (isClosed() || attr().get(FILL_COLOR) != null && attr().get(UNCLOSED_PATH_FILLED)) {
       if (path.contains(p)) {
```

Comment:
Line 177 and 178 can be merged into one line instead of two. This will increase the readabilty of the code. The line 177 and 178 can be merged as below.
eg: if ((isClosed() || attr().get(FILL_COLOR) != null) && (attr().get(UNCLOSED_PATH_FILLED)) && path.contains(p)) {}